app/models/floating_ip/operations.rb

```diff
@@ -0,0 +1,19 @@
+module FloatingIp::Operations
+  extend ActiveSupport::Concern
+
+  def update_floating_ip(options = {})
+    raw_update
```

Comment:
Once you pass `options` to `raw_update_floating_ip` you will not have that issue anymore.